it/stubs/FinancialDataStub.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package stubs
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import helpers.WireMockMethods
+import play.api.http.Status.{INTERNAL_SERVER_ERROR, OK}
+import play.api.libs.json.{JsObject, JsValue, Json}
+
+object FinancialDataStub extends WireMockMethods {
+
+  private def directDebitUri(vrn: String) = s"/financial-transactions/has-direct-debit/$vrn"
+
+  private def paymentUri(vrn: String) = s"/financial-transactions/vat/$vrn"
+
+  val directDebitJson: JsObject = Json.obj("directDebitMandateFound" -> false)
+  val errorJson: JsObject = Json.obj("FAILURE" -> "Oh dear")
+
+  private val paidTransactions: JsValue = Json.parse(
```

Comment:
This is confusing. The JSON here is two paid transactions - i.e. transactions that in reality would not be returned by the API call to retrieve payments due. Can you change a few things in the JSON here and rename this val to better illustrate the scenario. Refer to the stub data repo if you need examples of outstanding payments.